cosmo_tester/test_suites/test_security_ec2/ad_auth_test.py

```diff
@@ -0,0 +1,161 @@
+########
+# Copyright (c) 2015 GigaSpaces Technologies Ltd. All rights reserved
```

Comment:
I don't see a reason to put this test in a different module, since it is a security test and uses the same flow. maybe add it to the existing folder of tests?